src/mugs.js

```diff
@@ -295,7 +295,44 @@ define([
             rawControlXML: {
                 presence: 'optional',
                 lstring: 'Raw XML'
-            }
+            },
+            dataParent: {
+                lstring: 'Data Parent',
+                visibility: function(mug) {
+                    function recFunc(mug) {
+                        if (!mug) {
+                            return true;
+                        } else if (mug.__className === 'Repeat') {
+                            return false;
+                        }
+                        return recFunc(mug.form.tree.getNodeFromMug(mug).parent.value);
```

Comment:
It does now: [mugs.js:Mug.prototype.parentPath](https://github.com/dimagi/Vellum/pull/325/files#diff-fb1b9e010259d16ab20a8186e2d67b23R684)